src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -225,4 +231,39 @@ public String toString() {
                 + this.hasTableOfContents + ", hasCoversheets=" + this.hasCoversheets + ", hasFolderCoversheets="
                 + this.hasFolderCoversheets + ")";
     }
+
+    @Transient
+    public Integer getSubtiles(SortableBundleItem container, Map<BundleDocument, File> documents) {
+        return container
+                .getSortedItems().flatMap(SortableBundleItem::getSortedDocuments)
+                .map(i -> {
+                    try {
+                        return load(documents.get(i)).getDocumentCatalog().getDocumentOutline();
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    return null;
+                })
+               .filter(o->o!=null)
+               .filter(o->o.getFirstChild()!=null)
+               .mapToInt(o -> o!=null? getItemsFromOutline.apply(o) : 0).sum();
+    }
+
+    @Transient
+    Function<PDDocumentOutline,Integer> getItemsFromOutline =(outline) ->{
+        ArrayList<String> firstSiblings = new ArrayList<>();
+        PDOutlineItem anySubtitlesForItem = null;
+        if(outline!=null)
+        anySubtitlesForItem = outline.getFirstChild();
+
+        while (anySubtitlesForItem != null) {
+            firstSiblings.add(anySubtitlesForItem.getTitle());
+            anySubtitlesForItem = anySubtitlesForItem.getNextSibling();
+
+        }
+        if(firstSiblings.size()>0)
```

Comment:
I don't understand why this statement is here

Reply:
updated to
 private Function<PDDocumentOutline, Integer> getItemsFromOutline = (outline) ->{
        ArrayList<String> firstSiblings = new ArrayList<>();
        PDOutlineItem anySubtitlesForItem = null;
        if(outline != null) {
            anySubtitlesForItem = outline.getFirstChild();

            while (anySubtitlesForItem != null) {
                firstSiblings.add(anySubtitlesForItem.getTitle());
                anySubtitlesForItem = anySubtitlesForItem.getNextSibling();

            }
        }
        return firstSiblings.size();
    };